docs/get-started/installation.md

```diff
@@ -0,0 +1,159 @@
+  - Microsoft.iOS.Windows.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.Android.Sdk (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Sdk.BundleTool (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Ref (11.0.200-preview.4.245) installed.
+  - Microsoft.Android.Templates (11.0.200-preview.4.245) installed.
+  - Microsoft.iOS.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.iOS.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.iOS.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.MacCatalyst.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Sdk (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Ref (14.5.100-preview.4.638) installed.
+  - Microsoft.tvOS.Templates (14.5.100-preview.4.638) installed.
+  - Microsoft.macOS.Sdk (11.3.100-preview.4.638) installed.
+  - Microsoft.macOS.Ref (11.3.100-preview.4.638) installed.
+  - Microsoft.macOS.Templates (11.3.100-preview.4.638) installed.
+────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
+
+- Congratulations, everything looks great!
+```
+
+## Build and launch apps
+
+For instructions on how to create and build your first .NET MAUI app, see [Build your first .NET MAUI app](first-app.md).
+
+Once a .NET MAUI app has been created, it can be built using .NET CLI if required. The following .NET CLI commands show how to build and launch a .NET MAUI app on three different platforms:
+
+```dotnetcli
+dotnet build -t:Run -f net6.0-android
+dotnet build -t:Run -f net6.0-ios
+dotnet build -t:Run -f net6.0-maccatalyst
+```
+
+### iOS simulator selection
+
+It's possible to specify which simulator is launched and used for net6.0-ios by specifying the `_DeviceName` MSBuild property:
+
+```dotnetCLI
+dotnet build -t:Run -f net6.0-ios -p:_DeviceName=:v2:udid=<UDID>
+```
+
+You can retrieve a list of possible UDID values by executing the `simctl` list command:
+
+```console
+/Applications/Xcode.app/Contents/Developer/usr/bin/simctl list
+```
+
+## IDE support
+
+Currently, you should use the latest preview version of Visual Studio 2019 16.11 on Windows (with the Mobile development with .NET and Universal Windows Platform development workloads installed).
+
+Visual Studio for Mac support will arrive in a future release.
+
+### iOS from Visual Studio
+
+To build and debug .NET 6 iOS apps from Visual Studio 2019 you must manually install the .NET 6 SDK and iOS workloads on both Windows and macOS (your Mac build host).
+
+If, while connecting Visual Studio to your Mac through Xamarin Mac Agent (XMA), you are prompted to install a different version of the SDK, you can ignore the prompt since it refers to a legacy version of XMA.
+
+> [!NOTE]
+> Visual Studio 2019 can only currently deploy .NET MAUI iOS apps to the iOS simulator.
```

Comment:
What deployment targets are excluded by the "only .. to the iOS simulator" here?

Reply:
Physical devices. Updated in my branch.